index.js

```diff
@@ -15,34 +16,29 @@ const isLink = (object) => object && object.sys && object.sys.type === 'Link';
  */
 const findNormalizableLinkInArray = (array, predicate) => {
   if (!array) {
-    return;
+    return undefined;
   }
```

Comment:
We already ensure earlier that we have an array here, we might remove that check completely.

Reply:
We don't really ensure it beforehand, just by response convention, so I'd leave it in

Reply:
But, we do now since we pass an array with all possible entries to link to:
https://github.com/contentful/contentful-resolve-response/pull/9/files#diff-168726dbe96b3ce427e7fedce31bb0bcR91

Reply:
Sorry, I was checking on the wrong ref, without the code updates 🤦‍♂️